lib/nrf_modem_lib/nrf_modem_lib_trace.c

```diff
@@ -58,8 +165,9 @@ static bool rtt_init(void)
 }
 #endif
 
-int nrf_modem_lib_trace_init(void)
+int nrf_modem_lib_trace_init(struct k_heap *os_trace_heap)
```

Comment:
os_trace_heap ->  trace_heap 

Thats the same name used in the header file.

Reply:
Just noticed we have a trace_heap variable within the file. So perhaps change the trace_heap to os_trace_heap in the header file.